src/app/globals.css

```
@@ -12,9 +12,16 @@
 
   --color-star: hsl(32deg, 83%, 55%);
 
+  --color-success: hsl(142deg, 76%, 36%);
```

این حجم از دقت نیازه؟
نمیشد اینجوری نوشت؟
```suggestion
  --color-success: hsl(140deg, 75%, 35%);
```